Resources/Locale/en-US/cognislime/cognislime.ftl

```diff
@@ -0,0 +1,8 @@
+# Error messages
+cognislime-invalid = You can't use this on that.
```

Comment:
You can't make that sentient. would be better

Reply:
Done, thanks